src/jit/SimdX86Inl.h

```diff
@@ -940,6 +966,54 @@ static void simdEmitPMinMax(sljit_compiler* compiler, uint32_t operation, sljit_
     simdEmitSSEOp(compiler, is64 ? SimdOp::orpd : SimdOp::orps, rd, tmp);
 }
 
+static void simdEmitFloatMaxMinHelper(sljit_compiler* compiler, sljit_s32 rd, sljit_s32 rn, sljit_s32 rm, sljit_s32 tmp, SimdOp::Type opcode)
```

Comment:
Other functions use the following argument definition:
`(sljit_compiler* compiler, uint32_t operation, ...)`
So the opcode(s) follow(s) the compiler argument.